Package.swift

```diff
@@ -9,7 +9,7 @@
 import PackageDescription
 
 let package = Package(
-	name: "StORM",
+	name: "StORM-R",
```

Comment:
This is some of the problems I found using MySQL-StORM today. I didn't test it after Fork. Sorry, I made a mistake. I will continue to use the discovery of some existing problems